javav2/example_code/rekognition/src/main/java/com/example/rekognition/DescribeCollection.java

```diff
@@ -0,0 +1,70 @@
+// snippet-sourcedescription:[DescribeCollection.java demonstrates how to describe an Amazon Rekognition collection.]

```

Comment:
Does this apply a description to a collection or retrieve the description of a collection? If the former, I would add "how to add a description to an ...". For the latter, I would say "how to retrieve the description of an ...".

I would also rename the function to describe what it does. So either GetRekognitionCollectionDescription or SetRekognitionCollectionDescription.

Reply:
fixed